pkg/auth/service.go

```diff
@@ -260,8 +260,8 @@ func (s *DBAuthService) CreateUser(ctx context.Context, user *model.User) (int64
 		}
 		var id int64
 		err := tx.Get(&id,
-			`INSERT INTO auth_users (display_name, created_at, friendly_name, source) VALUES ($1, $2, $3, $4) RETURNING id`,
-			user.Username, user.CreatedAt, user.FriendlyName, user.Source)
+			`INSERT INTO auth_users (display_name, created_at, friendly_name, source, email, encrypted_password) VALUES ($1, $2, $3, $4, $5, $6) RETURNING id`,
```

Comment:
Verify what is the current database default - if it is empty string or null in case there is no email and/or password.
We like to keep the same values in create we invite or not a user.

Reply:
That indeed is the case, if an empty string is passed or nil, it is simply left blank as until now

Reply:
fixed